Core/src/org/sleuthkit/autopsy/texttranslation/TranslationOptionsPanel.java

```diff
@@ -16,7 +16,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.sleuthkit.autopsy.texttranslation.ui;
+package org.sleuthkit.autopsy.texttranslation;
```

Comment:
These change will mean that the options panel and its constructor are now exported in our public API.